www/about/policies/terms-of-service.md

```diff
@@ -35,94 +35,97 @@ or damage arising from your failure to keep your account secure.
-between those parties.
+  1. Establishing a Project on Gratipay involves an application and approval
+     process. As part of this process, the Gratipay community will have the
+opportunity to publicly evaluate and provide commentary about the Project.
+However, all decisions regarding whether to accept, reject, suspend, remove, or
+take any other action regarding a Project belong solely to Gratipay.
 
-1. Rules for Gratipay Teams
+  1. Gratipay reserves the right to reject, suspend, or remove a Project at any
+     time, and for any reason.
 
-  1. Teams may accept payments on the Service only for Open Work, meaning that
-     they provide a clear path for any individual to voluntarily begin
-contributing to the Team's work, and to share in any revenue the Team
-generates.
+1. Terms for Collaborators
 
-  1. A Team can only begin receiving payments from Participants once it has
-     provided a working withdrawal mechanism for receiving payments (e.g. bank
-account information).
+  1. Project Owners can use the Service to distribute payments to
+     Collaborators.
 
-  1. Teams and their Open Work must be consistent with Gratipay's [Brand
-     Guidelines](http://inside.gratipay.com/big-picture/brand/) and Acceptable
-Use Policy.
+  1. When you become a Collaborator, you agree that payments to you by the
+     Project Owner are made exclusively because of your collaboration in the
+Work described in the Project's profile.
 
-  1. Gratipay reserves the right to reject, suspend, or remove a Team at any
-     time, and for any reason.
+  1. Gratipay is not a party to any agreement between Project Owners and
+     Collaborators, and is not responsible to either the Project Owner or the
+Collaborator if either party breaches the terms of any such agreement (e.g. if
+the Collaborator fails to provide the promised goods or services).
 
-  1. Establishing a Team on Gratipay involves an application and approval
-     process. As part of this process, the Gratipay community will have the
-opportunity to publicly evaluate and provide commentary about the Team.
-However, all decisions regarding whether to accept, reject, suspend, remove, or
-take any other action regarding a Team belong solely to Gratipay.
+  1. Gratipay does not determine how payments are allocated and is not
+     responsible for how Project Owners distribute payments. The agreement
+between the Project's Owner and other Collaborators regarding the allocation of
+payments is solely between those parties.
+
+1. Terms for Givers
+
+  1. When you make a payment via a Project to a Project Owner, it is a payment
+     to fund the Work described in the Project's profile. You understand and
+acknowledge that your payment does not entitle you personally to receive any
+goods or services from the Project Owner, unless the Project Owner explicitly
+offers those things to Givers.
+
+  1. Gratipay is not obligated to provide refunds under any circumstance.
```

Comment:
I also might say "not obligated to provide refunds _, or produce Work,_ under any circumstances".